drivers/Makefile.dep

```diff
@@ -81,13 +81,13 @@ ifneq (,$(filter bm%280,$(USEMODULE)))
   USEMODULE += bmx280
 endif
 
-ifneq (,$(filter cc110x,$(USEMODULE)))
+ifneq (,$(filter cc110%,$(USEMODULE)))
+  USEMODULE += cc110x
+  USEMODULE += cc1xxx_common
   USEMODULE += ieee802154
```

Comment:
I know this was already in the past but since this is a complete rewrite I suggest to remove it. Not all applications use by default 802.15.4 so pulling the dependency will force the compilation of the module. I'd say just add it if the user explicitly needs it.

Reply:
I updated it to only pull `ieee802154` in when `gnrc_ipv6` is used. This will make sure that sixlowpan is used, if upper layers are added as well.

Here is the fixup addressing this: https://github.com/RIOT-OS/RIOT/pull/10340/commits/88fe887be356cb4b0f45db44ec28f4b8d1ccc1ea